absl/base/internal/sysinfo.cc

```diff
@@ -76,6 +76,12 @@ static int GetNumCPUs() {
 #if defined(_WIN32)
 
 static double GetNominalCPUFrequency() {
+// UWP apps don't have access to the registry and currently don't provide an
+// API informing about CPU nominal frequency.
+#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP) && \
+    !WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
```

Comment:
Do you know since which version of windows SDK is `WINAPI_FAMILY_PARTITION` defined? My only concern is that if it is not defined this will fail to compile.

Reply:
WINAPI_FAMILY_PARTITION has been defined in the Windows 8 SDK. This change has been successfully tested on Windows 7 with Win7 SDK using VS2008. This change works on Win7 because WINAPI_FAMILY_PARTITION is not defined in the Win7 SDK.